flake.nix

```diff
@@ -2,15 +2,11 @@
   inputs = {
     nixpkgs = {
       url = "github:nixos/nixpkgs/nixos-23.11";
-      # We use nixpkgs as input of `flakebox`, as it locks things like
```

Comment:
Had to revert it to actually get our nixpkgs updated, without needing to update flakebox all the time. I think it's OK.